### PR TITLE
Login Modal and Interview Form

### LINKED ISSUE #10: Add in `Add Interview` form for a candidates page
It should open below the candidate's information, so the interviewer would be able to refer to that information. 

Blocked by #9

### PR DESCRIPTION
**Pull request by meghatron3000**
Resolves #10

### CODE REVIEW

File: frontend/src/pages/interview.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+              <Label>
+                <Input type="radio" name="passion" /> 2 - Really into it, thinking about ways they
+                could contribute to the org
+              </Label>
+            </FormGroup>
+          </FormGroup>
+          <FormGroup>
+            <legend>Community</legend>
+            <Label>
+              Will they contribute to community or are they just using this as a resume booster?{' '}
```
Comment (tko22) [inline frontend/src/pages/interview.js]: whats the `{' '}` at the end?

File: frontend/src/pages/interview.js [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+import Link from 'next/link'
+
+type Props = {}
+class Interview extends Component<Props> {
+  constructor(props) {
+    super(props)
+    this.state = { value: '' }
+    this.handleSubmit = this.handleSubmit.bind(this)
+  }
+  handleSubmit(event) {
```
Comment (tko22) [inline frontend/src/pages/interview.js]: nit: fat arrows?? hehe

**Comment by tko22** [comment]
Remember to Squash and Merge + include issue number in commit message

image

File: frontend/src/pages/interview.js (hunk 2 above)
Comment (meghatron3000) [reply frontend/src/pages/interview.js]: oh shoot is that function being used?
let me check

File: frontend/src/pages/interview.js (hunk 1 above)
Comment (meghatron3000) [reply frontend/src/pages/interview.js]: oh that's just a space for now ( it makes things more readable before we add css)

**Pull request merged**